exporters/metrics/README.md

```diff
@@ -15,13 +13,21 @@ Google Cloud Monitoring is a managed service provided by Google Cloud Platform.
 
 ## Usage
 
-TODO(jsuereth): Write this.
-
 See [the code example](../../examples/metrics) for details.
 
 ## Authentication
 
-TODO(jsuereth): Write this section.
+If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instructions in Authentication.
+
+This exporter uses [google-cloud-java](https://github.com/googleapis/google-cloud-java), which uses the [google-auth-library-java](https://github.com/googleapis/google-auth-library-java) for authentication. For details about how to configure the authentication see [here](https://github.com/googleapis/google-cloud-java#authentication).
+
+If you prefer to manually set the credentials, they can be passed to the exporter using the [`setCredentials`](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/7a8dcf420f5551bf0df969f83928b18c443da5aa/exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricConfiguration.java) method:
+```java
+MetricConfiguration.builder()
+    .setCredentials(new GoogleCredentials(new AccessToken(accessToken, expirationTime)))
```

Comment:
Is this something users might do? The access token is different from credential.

If they just want to point at an exported JSON service account cred, the client google auth lib should work with `GOOGLE_APPLICATION_CREDENTIALS` env

Reply:
This is certainly a rare use-case. I'm documenting it here since it was also documented for trace exporter.